sdk/communication/communication-job-router/src/jobRouterClient.ts

```diff
@@ -318,12 +330,11 @@ export class JobRouterClient {
     return this.client.jobRouter.declineJobAction(workerId, offerId, options);
   }
 
-  // Worker Actions
   /**
    * Creates a worker.
    * Returns the registered worker.
-   * @param workerId - The ID of the worker to create.
-   * @param options - Create worker options.
+   * @param workerId - The id of the worker to create.
+   * @param options - Options for creating a router worker. Uses merge-patch semantics: https://datatracker.ietf.org/doc/html/rfc7386.
```

Comment:
We can remove the "merge patch semantics" comment from any Create method, since it's only relevant for updates.